Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
I tried adding it and the TeamCity build process complained.  Once merged I'll fast track a Linux update so I can test it independently.